CHANGELOG-2.1.md

```diff
@@ -1,3 +1,13 @@
+# Changelog since v2.0.1
+
+## Bug Fixes
+
+### Snapshot Controller
+
+- Cherry pick PR #293: Disallow a pre-provisioned VolumeSnapshot pointing to a dynamically created VolumeSnapshotContent. ([#303](https://github.com/kubernetes-csi/external-snapshotter/pull/303), [@yuxiangqian](https://github.com/yuxiangqian))
```

Comment:
do you want to also include issue #63

Reply:
We didn't do anything with that issue in this release?  That was done as part of 2.0 when snapshot was promoted to beta.

Reply:
urrrr. I must have typed some weird number
I meant to refer to issues? #290 #291 #292, though they could be also found in PR #293, I feel its more clear what have been fixed.
or at least should include them in ## Bug Fixes session.

Reply:
I didn't add issue numbers because those were not in other changelog, but I can add them if you prefer.

Added them now.